src/components/header/styles/modal-account-unification.scss

```diff
@@ -0,0 +1,201 @@
+@import 'src/css/quasar.variables.scss';
+@import 'src/css/utils.scss';
+
+.btn {
+  width: 100%;
+  font-size: 20px;
+  font-weight: 600;
+  height: 52px;
+  @apply tw-rounded-full;
+  &.close {
+    background: transparent;
+    color: $navy-1;
+    border: solid 1px $navy-1;
+  }
+}
+
+.account-name {
+  @apply tw-font-bold tw-flex tw-items-center tw-justify-start tw-text-base;
```

Comment:
Sorry but we are not using Tailwind CSS anymore, please use pure CSS or SCSS instead.

Reply:
Replaced all tailwind styles to pure css